docs/index.md

```diff
@@ -1,41 +1,74 @@
+```eval_rst
```

Comment:
This file seems like there might have been a rebasing issue, but it looks like it's right in the end, so NBD.

Reply:
Nevermind. I see that this was moved from index.rst